### DISCUSSION TITLE
快速输入 BUG

### DISCUSSION THREAD

**Discussion by aaronameixa** [densen2014/docs#36] [Q&A]
![image](https://github.com/densen2014/docs/assets/122285019/47124c5f-3646-43ab-9a97-2bbe9de87964)

**Comment by aaronameixa** [comment]
版本 2023-10-24

**Comment by densen2014** [comment]
先咨询一下客服用法

**Reply by aaronameixa** [reply]
重新安装之前的版本 2023-04-12
